identity-account/README.md

```diff
@@ -0,0 +1,70 @@
+# Iota Identity Account
+
+The Account is an interface for creating and managing identities on the Iota Tangle. It includes easy-to-use functions that allow many operations including:
+
+- Creating an identity or resolving an existing DID Document.
+- Updating a DID Document by adding or removing:
+  - Services.
+  - Verification Methods.
+  - Verification Relationships.
+  - Custom properties.
+- Managing and storing the state of DID Documents.
+- Managing and storing private encryption keys used as verification methods.
+
+## Account Creation
+
+Creating an account is done through the `AccountBuilder`.
+
+```rust
+  let account: Account = Account::builder()
+    .create_identity(IdentitySetup::default())
+    .await?;
+```
```

Comment:
This will run on `cargo test` and might occasionally fail because of some io error. One can work around this with the [Easy doc initialization pattern](https://rust-unofficial.github.io/patterns/idioms/rustdoc-init.html). 

```suggestion
use identity_account::Account;
use identity_account::Result;
 
async fn build_account() -> Result<Account> {
    let account: Account = Account::builder()
    .create_identity(IdentitySetup::default())
    .await?;
    Ok(account)
```

This way the `cargo test` checks that the code compiles, but it is not run.